roles/xdebug/templates/xdebug.ini.j2

```diff
@@ -0,0 +1,47 @@
+; {{ ansible_managed }}
+
+[XDebug]
+zend_extension="xdebug.so"
+
+; Remote Debugging
+xdebug.remote_enable={{ xdebug_remote_enable }}
+xdebug.remote_connect_back={{ xdebug_remote_connect_back }}
+xdebug.remote_host={{ xdebug_remote_host }}
+xdebug.remote_port={{ xdebug_remote_port }}
+xdebug.remote_handler="dbgp"
+xdebug.remote_log={{ xdebug_remote_log }}
```

Comment:
I think this string value should probably have quotes:
`xdebug.remote_log="{{ xdebug_remote_log }}"`